src/samples/ChildBenefitsClaim/index.tsx

```diff
@@ -131,12 +131,23 @@ export default function ChildBenefitsClaim() {
 
   const [inprogressClaims, setInprogressClaims] = useState([]);
   const [submittedClaims, setSubmittedClaims] = useState([]);
+  const [showBeginNewClaimButton, setShowBeginNewClaimButton] = useState(false);
 
   function doRedirectDone() {
     history.replace('/');
     // appName and mainRedirect params have to be same as earlier invocation
     loginIfNecessary({ appName: 'embedded', mainRedirect: true });
   }
+  useEffect(() => {
+    if (submittedClaims[0]?.HideBeginClaimSubmitted === true) {
+      setShowBeginNewClaimButton(true);
+    } else if (
+      submittedClaims[0]?.HideBeginClaimSubmitted === false ||
+      submittedClaims[0]?.HideBeginClaimSubmitted === null
+    ) {
+      setShowBeginNewClaimButton(false);
+    }
+  });
```

Comment:
```suggestion
  }, []);
```
don't forget the dependency array

Reply:
I want the code inside the useEffect to run on every render.

Reply:
Could we perhaps pass in submittedClaims as a dependency so that when ever the they change we decide if to show the button?

Reply:
done